docs/docs/integrations/providers/databricks.md

```diff
@@ -103,14 +103,31 @@ See [MLflow LangChain Integration](/docs/integrations/providers/mlflow_tracking)
 
 SQLDatabase
 -----------
-You can connect to Databricks SQL using the SQLDatabase wrapper of LangChain.
-```
-from langchain.sql_database import SQLDatabase
+You can connect to Databricks SQL or query structured data See [Structured retrieval AI agent tools](https://docs.databricks.com/en/generative-ai/agent-framework/structured-retrieval-tools.html#table-query-tool).
 
-db = SQLDatabase.from_databricks(catalog="samples", schema="nyctaxi")
 ```
+from databricks_langchain.uc_ai import DatabricksFunctionClient
```

Comment:
I might just leave this out, it'll be in the linked docs page (it'd be nice not to have to duplicate + keep content up to date across places)